posthog/models/constance.py

```diff
@@ -0,0 +1,46 @@
+import json
+from contextlib import contextmanager
+from typing import Any
+
+from django.db import models
+
+from posthog.settings import CONSTANCE_CONFIG, CONSTANCE_DATABASE_PREFIX
+
+
+class Constance(models.Model):
```

Comment:
do we really want to name this the same thing as an existing well-known library?

Reply:
Why not?

Reply:
Yes, I was a bit confused at first because I thought this just builds on top of the `constance` library. If this is our own mechanism, let's call it something different, maybe just literally e.g. `InstanceSetting`